docs/starters.yml

```diff
@@ -6067,6 +6067,22 @@
     - Pagination
     - SEO
     - Phone browser Support
+- url: https://dropinblog-gatsby-starter.netlify.app/
+  repo: https://github.com/DynamisDevelopment/dib-gatsby-starter
```

Comment:
repo link is an 404

is it this here https://github.com/DropInBlog/gatsby-starter-dropinblog ?

Reply:
You're right, and it was fixed in #24151.